File: CMS/src/app/shared/CompanyGroup.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { CompanyGroup } from 'src/app/shared/CompanyGroup.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class CompanyGroupService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "CompanyGroup";
    }
}

File: Service/Implement/CompanyGroupService.cs
namespace Service.Implement
{
    public class CompanyGroupService : BaseService<CompanyGroup, ICompanyGroupRepository>
    , ICompanyGroupService
    {
    private readonly ICompanyGroupRepository _CompanyGroupRepository;
    public CompanyGroupService(ICompanyGroupRepository CompanyGroupRepository) : base(CompanyGroupRepository)
    {
    _CompanyGroupRepository = CompanyGroupRepository;
    }
    }
    }

